test/distributed/org/apache/cassandra/distributed/test/trackwarnings/TombstoneWarningTest.java

```diff
@@ -53,6 +54,9 @@
 import org.apache.cassandra.service.QueryState;
 import org.apache.cassandra.service.reads.trackwarnings.CoordinatorWarnings;
 import org.assertj.core.api.Assertions;
+import org.assertj.core.api.Condition;
+
+import static org.assertj.core.api.Assertions.assertThat;
 
 public class TombstoneWarningTest extends TestBaseImpl
```

Comment:
nit: There are 2 `TombstoneWarningTest` classes in the codebase...might make sense for the other one to be `TombstoneCompactionTest` or something similar.

Reply:
I can rename this TombstoneCountWarningTest